tests/std/tests/P1023R0_constexpr_for_array_comparisons/test.cpp

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <array>
+#include <cassert>
+
+using namespace std;
+
+constexpr bool test_operator_eq() {
+    array<int, 5> a0{2, 8, 9, 1, 9};
+    array<int, 3> a1{2, 8, 9};
+    array<int, 5> a2{2, 8, 9, 1, 8};
+    array<int, 0> a3({});
+
+    assert(a0 == a0);
+    assert(a1 == a1);
+    assert(!(a0 == a2));
+    assert(a3 == a3);
+
+#ifdef __cpp_lib_to_array

```

Comment:
`__cpp_lib_to_array` is always defined in C++20 mode, and this test only runs in C++20 mode, so this need not be conditional. That said, these conditionally compiled sections seem to be adding test coverage for `to_array` moreso than the comparison operations.

Reply:
Would you suggest removing this entirely then?